app/src/main/res/values/strings.xml

```diff
@@ -9,4 +9,7 @@
     <string name="orange_status_title">Uważaj na siebie</string>
     <string name="orange_status_content">Prawdopodobnie przebywałeś w miejscach, w których znajdowały się osoby zarażone. Zachowaj ostrożość i ogranicz wyjścia do minimum. Po więcej informacji kliknij [tutaj].</string>
     <string name="placeholder_content">- Ograniczenia w przemieszczaniu się z wyjatkiem drogi do pracy, wolontariatu, załatwiania spraw niezbędnych do życia codziennego.\n- W autobusie może być tyle osób ile wynosi połowa liczby miejsc siedzących.\n- Całkowity zakaz zgromadzeń - chyba, że z najbliższymi. Więcej informacji [tuta].</string>
+    <string name="missing_email_client">Błąd uruchomienia klienta poczty.</string>
```

Comment:
Czy te teksty już się znajdują w pliku z tłumaczeniami?

Reply:
W komentarzu w kodzie napisałem, że to dodatkowe teksty i trzeba poprosić o copy.

Reply:
No tak, ale lepiej już dodać do pliku identyfikatory i je oznaczyć jako brakujące/tymczasowe

Reply:
lint nam zablokuje release, jeśli nie dodamy tego też w values-pl.